internal/crossplane/client.go

```diff
@@ -200,6 +200,31 @@ func (cp Crossplane) Instance(ctx context.Context, id string, plan *Plan) (inst
 	return inst, true, nil
 }
 
+// FindInstanceWithoutPlan is used for retrieving an instance when the plan is unknown.
+// It needs to iterate through all plans and fetch the instance using the supplied GVK.
+// There's probably an optimization to be done here, as this seems fairly shitty, but for now it works.
```

Comment:
We can just do a `get` with the UID as name for each service this instance serves. We can get the GVK of the first plan we find of a service since it's the same for all plans of this service.